src/components/ReportCard.js

```diff
@@ -70,69 +94,59 @@ const StyledPill = styled.div`
   display: flex;
   justify-content: center;
   align-items: center;
-  width: 69px;
-  height: 28px;
+  width: 70px;
+  height: 30px;
   border-radius: 20px;
-  position: absolute;
-  left: 25px;
-  top: 25px;
-
-  ${minBreakpointQuery.medium`
-    left: 25px;
-    top: 25px;
-  `}
-
-  ${minBreakpointQuery.large`
-    left: 40px;
-    top: 40px;
-  `}
 `;
 
 const StyledImage = styled.img`
+  display: grid;
+  grid-area: 1 / 1 / 1 / 1;
+  position: relative;
+  border-radius: 15px;
   max-height: 100%;
```

Comment:
These three styles break the card when more text is added not allowing the image to grow.

![Screenshot 2024-02-14 at 14 48 09](https://github.com/tomkeyes-attachdigital/DiversityTravelHomePage/assets/60260005/42ae31f6-36cc-4b7d-aa5f-00009b0acc75)